docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -346,6 +454,28 @@ Before executing an action, TestCafe waits for the target element to appear
 in the DOM and become visible. If this does not happen
 within the [selector timeout](#selector-timeout), the test fails.
 
+Note that if selector initializer has several matching DOM nodes on the page the action will be executed for the first one from the set.
+
+## Using Selectors to Define Assertion Actual Value
+
+You can check is a DOM node has an expected state by passing [snapshot API shorthand](#snapshot-api-shorthands) directly to [test actions](../assertions/index.md).
```

Comment:
You can check whether a particular DOM node has an expected state by passing [snapshot API shorthand](#snapshot-api-shorthands) directly to [test actions](../assertions/index.md).